x-pack/test/detection_engine_api_integration/basic/tests/update_rac_alerts.ts

```diff
@@ -1,159 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Wasn't this test to track updates from 7.x to 8.x? Wouldn't we want to keep this around or maybe modify it until we're sure most people are off 7.x? I know we're still making use of `RAC_ALERTS_BULK_UPDATE_URL` in the code in `useUpdateAlertsStatus`

Reply:
Not any more, actually. You can check it in the file x-pack/plugins/security_solution/public/common/components/toolbar/bulk_actions/use_update_alerts.ts 
I removed this usage, because it was legacy.